design/src/main/kotlin/uk/govuk/app/design/ui/component/Header.kt

```diff
@@ -46,7 +49,8 @@ fun ChildPageHeader(
     text: String? = null,
     onBack: (() -> Unit)? = null,
     onAction: (() -> Unit)? = null,
-    actionText: String? = null
+    actionText: String? = null,
+    focusRequester: FocusRequester? = null
```

Comment:
Can we not do this via the modifier we pass in i.e. request focus for the whole header (that way this very specific code would live where it is most relevant)? Rather than passing in the focus requester and applying it to only the title?

Reply:
@davidc-gds yeah, that's what I did initially (made the ChildPageHeader focusable and set focus on it) but it didn't take focus. I also tried merging descendants but then the whole ChildPageHeader is focussed. This was only solution I could find.

Reply:
TBH in that case I think it would make more sense to use a custom label for the title on the visited screen rather than add that functionality to the child page header. We could still use the child page header for the back button and action button, just not give it a title. Still not perfect but similar approach to topics and subtopics (not scrollable though). What do you think @TimGJD @gclssvglx ?

Reply:
@davidc-gds personally, I think the TalkBack issue is super edge-case and can be mostly resolved by using a delay. Using components on the other hand has got to be the preferred long-term approach and is more sustainable than edge-case fixes for what are essentially TalkBack bugs.  So, on balance my preference would be to keep the ChildPageHeader and add just the delay. Log it as a known issue and hope that it's resolved where is should be - in TalkBack.

My concern is that if we keep putting patch after patch in, the code will start to get unmaintainable.